test/test_product_config.py

```diff
@@ -494,11 +506,44 @@ def test_from_config_narrowband(
         assert acv.data_rate(1.0, 0) == 27392e6 * 2 / 8
         assert acv.input_labels == narrowband_config["src_streams"]
         assert acv.w_cutoff == 1.0  # Default value
-        assert acv.window_function is None
+        assert acv.window_function == defaults.WINDOW_FUNCTION
         assert acv.pfb_taps == defaults.PFB_TAPS
         assert acv.dither is None
         assert acv.command_line_extra == []
 
+    def test_from_config_narrowband_vlbi(
+        self,
+        narrowband_vlbi_config: Dict[str, Any],
+        src_streams: List[DigBasebandVoltageStreamBase],
+    ) -> None:
+        acv = GpucbfAntennaChannelisedVoltageStream.from_config(
+            Options(), "vlbi1_acv", narrowband_vlbi_config, src_streams, {}
+        )
+        # Most attributes we don't bother testing - just those affected by VLBI mode
+        assert acv.bandwidth == src_streams[0].adc_sample_rate / 2 / 8
+        assert (
+            acv.n_samples_between_spectra
+            == 2
+            * narrowband_vlbi_config["n_chans"]
+            * narrowband_vlbi_config["narrowband"]["decimation_factor"]
+        )
+        assert acv.data_rate(1.0, 0) == 27392e6 * 2 / 8
```

Comment:
You're using `/ 8` 
* in the `bandwidth` check,
* the `["decimation_factor"]` (it's `8` in the config fixture), and
* in the `data_rate(...)` check

Even if we ignore the middle example, the first two are ambiguous enough. At least to me, anyway. I thought I was sure what unit `bandwidth` is in but now not so much.

Reply:
I've introduced a named constant for this repeated magic value. Arguably I should do it for various other constants (like 1712e6), but maybe this one is more confusing than others.